specs/derivation.md

```diff
@@ -974,46 +979,171 @@ Some network upgrades require automated contract changes or deployments at speci
 To automate these, without adding persistent changes to the execution-layer,
 special transactions may be inserted as part of the derivation process.
 
-#### Eclipse: L1Block predeploy upgrade
+#### Ecotone
 
-The `L1Block` contract is upgraded to process the new Eclipse L1-data-fee parameters and L1 blob base-fee.
+The block before the Ecotone hardfork, contains the following transactions in this order:
 
-The `L1Block` is called in the very first transaction of the block,
-and parsed to retrieve the L1 block attributes.
+- L1 Attributes Transaction
+- User deposits from L1
+- Network Upgrade Transactions
+  - L1Block deployment
+  - GasPriceOracle deployment
+  - L1Block proxy update
```

Comment:
Could we make this more specific by saying ERC -1967 implementation slot modified

Reply:
Will do 👍